### ISSUE TITLE
Supported Nested groups

### ISSUE THREAD

**Issue by ujamer** [jgwhite/ember-sortable#27]
Hi! 
  In my use case I have a sortable set nested inside another array of objects. Something like:

```js
model() {
  return {
    groups: [
      { 
        id: 0,
        items: [
          { name: 'Uno' },
          { name: 'Dos' },
          { name: 'Tres' },
          { name: 'Cuatro' },
          { name: 'Cinco' }
        ]
      },{ 
        id: 1,
        items: [
          { name: 'Eins' },
          { name: 'Zwei' },
          { name: 'Drei' },
          { name: 'Vier' },
          { name: 'Fünf' }
        ]
      }
    ]
  };
}
```

In the current code, I cannot specify which group the `reorderItems` action handler should target, since only the `newOrder` array is given. 

Could we modify the `sortable-group` component to accept an additional `source` variable, so that the `reorderItems` know which group it is suppose to modify?

I've created a branch [here](https://github.com/ujamer/ember-sortable/tree/support-nested-group).

**Comment by rsaccon** [comment]
+1
I also have such a use case

**Comment by jgwhite** [comment]
I’ve been considering adding a `model` attr to `sortable-group` as proposed in #22. Would that suit your needs?

**Comment by rsaccon** [comment]
I think so

**Comment by jgwhite** [comment]
@rsaccon cool. I’m hoping to put a bit of time into this soon but if anyone would like to give it a try first, I would welcome a pull request :smile:

**Comment by ujamer** [comment]
Sure. I've opened a pull request #28.

**Comment by jgwhite** [comment]
Closed by #28
